Add Less Maddening Shivering Isles Fetch Quests

Sabre142 opened the pull request.

Sabre142 committed:
Add Less Maddening Shivering Isles Fetch Quests

* Add mod entry
 - To the Items section.
 - Places items required for The Antipodean Hammer
   and The Museum of Oddities quests in the SI world.

* Add location
 - https://www.nexusmods.com/oblivion/mods/52621/

* Add requirement
 - Missing SI master.

* Add tag suggestion
 - Invent.Add, adds items to a container in 00066F19.

* Add cleaning data
 - Version 1.0.

sibir-ine reviewed (changes requested):
The **Items** section is more for mods that add new items (e.g., armor, weapons). **Gameplay - Quests** would be more appropriate.

Sabre142 committed:
Add Less Maddening Shivering Isles Fetch Quests

* Add mod entry
 - To the Gameplay - Quests section.
 - Places items required for The Antipodean Hammer
   and The Museum of Oddities quests in the SI world.

sibir-ine reviewed (approved):
Thanks!
